### PR TITLE
Final push

### PR DESCRIPTION
**Pull request by dfsalam**
I´m adding the final versions of index.html and styles.css.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,24 +12,30 @@
...
+    <div class="status-bar-space"></div>
+    <div class="status-images">
+      <img src="./Images/status.png">
+    </div>    
   </div>
-  <nav class="navbar">
-     
+  <nav class="navbar">     
     <div class="nav-name">Jhon Doe</div>
-    <div class="nav-menu"></div>
```
Comment (iambenkis) [inline index.html]: - [ ] Kindly note that your logo should be made a link as it is usually used to navigate back to the home page when someone has navigated to other pages. Changing it like below will fix it.
```
        <h1 class="logo-txt"><a href="#">My Logo</a></h1>
```

- [ ] Also the logo should not be inside a ```div``` kindly put it in a ```hx``` tag.

File: index.html [hunk 2]
```diff
@@ -12,24 +12,30 @@
...
-  </div>
-    
+    <div class="icons">
+      <img src="./Images/Icon -GitHub.png">
+      <img src="./Images/Icon - Linkedin.png">
+      <img src="./Images/Icon - Angelist.png">
+      <img src="./Images/Icon - Twitter.png">
+      <img src="./Images/Icon - Medium.png">     
+    </div>
+  </div>
```
Comment (iambenkis) [inline index.html]: - [ ] Great job @dfsalam  you have very clean markup :+1: However, according to the [project's requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md), this section should not be inside a `<div>` tag but a `<section>` tag. Kindly change the `<div>` to `<section>` an this should be good.

File: index.html [hunk 3]
```diff
@@ -12,24 +12,30 @@
...
   <div class="container">
     <div class="primary-text">Hey There. I’m Alex</div>
     <div class="supporting-text">I’am a Software Developer</div>
     <div class="description">I can help you build a product , feature or website. 
       Look through some of my work and experience! If you like what you see and have a project you need coded, 
       don’t hesitate to contact me.</div>
-  </div>
-    
+    <div class="icons">
+      <img src="./Images/Icon -GitHub.png">
```
Comment (iambenkis) [inline index.html]: - [ ] Let's kindly try to stick to the design as much as possible. The "the margin between the description and the icons should be increased". Kindly change this.

Your design :

Screenshot from 2022-12-01 09-34-11

The Figma Design : 

Screenshot from 2022-12-01 09-34-35

**Review by iambenkis — CHANGES_REQUESTED** [review]
## Hello @dfsalam   :wave: , 
I'm @iambenkis your reviewer for this PR :man_beard: 

Good job setting your portfolio you have done a tremendous job :clap:  :clap:  
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] Good job in implementing  your project according to the Figma design.However, you add something that should not be there. The part in **red** represent the device you are you, it in this case a smart phone :iphone:. It's not supposed to be a part of the design. Kindly remove that part.

Screenshot from 2022-12-01 08-41-11

- [ ] Kindly note that you are supposed to stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as much as possible. The header is supposed to fill all the screen without a margin or a padding. Kind work on that and take a look at the picture below. :eyes: 

Screenshot from 2022-12-01 09-14-17


_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nemwel-Boniface** [comment]
Hello @iambenkis why am I tagged in this PR?

**Comment by iambenkis** [comment]
@Nemwel-Boniface , I’m sorry I use a template that is your! Kindle accept apologies 🙏

File: styles.css [hunk 4]
```diff
@@ -1,43 +1,46 @@
...
-  height: 590px;
-  background: #1C1A19;  
+  height: 100vh;
+  background: #1c1a19;
   background-image: url('./Images/Background.png');
   background-repeat: no-repeat;
-  background-size:initial;
+  background-size: initial;
   background-attachment: fixed;
   background-position: bottom right;
```
Comment (Tufahel) [inline styles.css]: - [ ] ***[OPTIONAL]***
  - Kindly consider removing `(background-attachment: fixed;)`. It is creating problems in reading the paragraph.

- Screenshot
new

- GIF
ezgif com-gif-maker (1)

File: index.html [hunk 5]
```diff
@@ -11,25 +11,24 @@
...
+    <div class="nav-space"></div>
+    <div class="nav-menu"><img src="./Images/menu.png" height="30px"></div>        
   </nav>
-  <div class="container">
+  <section class="container">
     <div class="primary-text">Hey There. I’m Alex</div>
     <div class="supporting-text">I’am a Software Developer</div>
     <div class="description">I can help you build a product , feature or website. 
       Look through some of my work and experience! If you like what you see and have a project you need coded, 
       don’t hesitate to contact me.</div>
```
Comment (Tufahel) [inline index.html]: ***[OPTIONAL]*** Kindly use specific tags for specific sections.

- Such as: 
  - For the title `Hey There. I'm Alex`  you could use `h1` tag.
  - For `I'm a Software Developer` you could use `h2` or `h3` tags.
  - For a paragraph, it's better to use a `p` tag inside the `div` tag.

- Reason behind this is, this way will be semantically correct and the screen readers and other technologies will thank you.

- Here is a [reference](https://stackoverflow.com/questions/28529390/ok-to-have-text-in-div-without-paragraph-tag) from Stackoverflow about why it's better to use a specific tag for a specific section. You can consider checking it.

- `Though it is also an important issue, Kindly fix it before merging.`

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @dfsalam, 

Alt Text

Great work on implementing previous reviewer suggestions! :+1: 
There are some issues that you still need to work on to go to the next project, but you are almost there!

## Highlights 🌟

✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.

## Required Changes ♻️
- [ ] As per previous reviewer suggestion, kindly note that you are supposed to stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) as much as possible. The header is supposed to fill all the screen without a margin or a padding. Kind work on that and take a look at the picture below.

Screenshot from 2022-12-01 23-32-37
  
  - I suggest you to add `margin: 0` in CSS `body` property. Such as ` body { margin: 0; } `.

_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -11,25 +11,24 @@
...
-  </div>
-    
+      don’t hesitate to contact me.</p>
+    <div class="icons">
+      <img src="./Images/Icon -GitHub.png">
+      <img src="./Images/Icon - Linkedin.png">
+      <img src="./Images/Icon - Angelist.png">
+      <img src="./Images/Icon - Twitter.png">
+      <img src="./Images/Icon - Medium.png">     
+    </div>
```
Comment (Kamba56) [inline index.html:31]: - [Optional] I think the list of social icons should be an unordered list. This is best for HTML semantics. The list should look like 
```html
<ul>
  <li>
    <a><img></a>
  </li>
  <li>
    <a><img></a>
  </li>
</ul>
```

**Review by Kamba56 — APPROVED** [review]
Hi @dfsalam , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**